modules/ROOT/pages/editor-premium-upgrade-promotion.adoc

```diff
@@ -0,0 +1,26 @@
+= Premium upgrade promotion
+:navtitle: {productname} Premium upgrade promotion
+:description: Editor options related to turning the Premium promotion display off
+:keywords:
+
+== Premium upgrade promotion defaults
+
+When {productname} 6.2 or later is running in self-hosted Community mode, an *Upgrade* promotion button appears in the top-right of the editor menu bar by default. The button does note appear when {productname} is running in Community mode on the Tiny cloud.
+
+This promotion button redirects to a link:{companyurl}/tinymce-self-hosted-premium-features/[promotion page], arguing the benefits of Community mode users switching from self-hosted to the {cloudname}.
```

Comment:
```suggestion
This promotion button redirects to a link:{companyurl}/tinymce-self-hosted-premium-features/[promotion page], showcasing the benefits of our wide variety of xref:plugins#premium-plugins[Premium Plugins] available for {productname}.
```
I wrote the wrong thing without reading the page, looks like the page mostly talks about premium plugins, not the cloud. Since the promotion only appears to self-hosted users on the core (free) plan, many of whom are from the open-source community, there is benefit in showcasing our premium features, without necessarily encouraging the switch to cloud.